.circleci/config.yml

```diff
@@ -68,6 +68,24 @@ jobs:
             docker build -t ld4p/sinopia_profile_editor:latest .
             echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
             docker push ld4p/sinopia_profile_editor:latest
+            docker push ld4p/sinopia_profile_editor:dev
+
+  register_tag_image:
+    <<: *defaults
+    steps:
+      - checkout
+      - setup_remote_docker
+      - restore_cache: # gives us back matching node_modules
+          key: dependency-cache-prod-{{ checksum "package.json" }}
+      - attach_workspace:
+          at: dist
+      - run:
+          name: Build & Register Image
+          command: |
+            docker build -t ld4p/sinopia_profile_editor:latest .
+            echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
+            docker push ld4p/sinopia_profile_editor:$CIRCLE_TAG-stage
```

Comment:
I'm not sure if this will work. It looks like you're building latest, but pushing a tagged image. If I'm understanding correctly that we don't have different versions for stage and prod, then can do:
```
docker build -t ld4p/sinopia_profile_editor:$CIRCLE_TAG .
echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
docker push ld4p/sinopia_profile_editor:$CIRCLE_TAG
```